src/cache.cc

```diff
@@ -17,7 +17,7 @@ namespace std {
     seed ^= hasher(v) + 0x9e3779b9 + (seed << 6) + (seed >> 2);
   }
 
-#if defined(__x86_64__) || defined(__i386__)
```

Comment:
I would limit it here just to __ppc64__ and __ppc__  for the sake of  consistency, see  https://sourceforge.net/p/predef/wiki/Architectures/

Reply:
@andriish Can you elaborate on what you mean by "consistency"? I'm not sure what you're trying to communicate and the link isn't informative. I'm sure to you it is clear, but I'm going to need you to write out your point.